addons/event_sale/models/event.py

```diff
@@ -6,6 +6,35 @@
 from openerp.osv import fields as old_fields
 
 
+class event_type(models.Model):
+    _inherit = 'event.type'
+    _inherits = {"product.product": "product_id"}
+
+    product_id = fields.Many2one('product.product', 'Product', ondelete='cascade', required=True, domain=[('event_ok', '=', True), ('type' '=' 'service')])
```

Comment:
I don't get it ... you creata an event type "Seminar". It creates a product. With a price. That means that all seminars will be based on the same product, whatever the event ?

Hugh.